config/routes.rb

```diff
@@ -89,6 +89,8 @@
   get '/popular/recent' => 'likes#recent'
 
   get '/upload' => 'spectrums#new'
+  post '/spectrums/create' => 'spectrums#create'
+  post '/comments/spectrum/:id' => 'comments#spectrum'
```

Comment:
comment post ajax call gave path not found error

Reply:
Great!